### ISSUE TITLE
How to avoid the false death state of inference? Is there a timeout setting?

### ISSUE THREAD

**Issue by dnlzsy** [FunAudioLLM/CosyVoice#426]
How to avoid the false death state of inference? Is there a timeout setting?
How to avoid the reasoning hang state, is there a timeout setting?
When encountering special text, the reasoning often hangs. Now I deal with it one by one, I don’t know if I can set a timeout and then handle the exception.

**Comment by aluminumbox** [comment]
please provide an example when inference hangs

**Comment by aluminumbox** [comment]
try set min_token_text_ratio=0  https://github.com/FunAudioLLM/CosyVoice/blob/main/cosyvoice/cli/model.py#L93

**Comment by dnlzsy** [comment]
Yes, when the number of characters is less than 3, it will hang there. I will try to set this parameter.
